zipwalk/__init__.py

```diff
@@ -22,7 +22,7 @@ def zipwalk(file: ZipFile, suffixes: set = None) -> list:
 
     files = {i.filename for i in infos} - (zips | dirs)
 
-    yield file, zips, files
+    yield file, sorted(list(zips)), sorted(list(files))
```

Comment:
Do we need to convert sets to lists before sorting them? I don`t think so...

Testing it...

```py
>>> sorted({5, 1, 4, 3})
[1, 3, 4, 5]
```

Nope.

```suggestion
    yield file, sorted(zips), sorted(files)
```
```

Reply:
So concise, I love it!